test/progress.ts

```diff
@@ -72,7 +72,7 @@ test('download progress', withServer, async (t, server, got) => {
 
 	const events: Progress[] = [];
 
-	const {body} = await got({encoding: null})
+	const {body} = await got<Buffer>({encoding: null})
```

Comment:
Note: https://github.com/sindresorhus/got/commit/6cc3d9f3027f28ce4dcbe8605d104e65f7e66787

Reply:
Merged and fixed 🤞